dataduct/config/config.py

```diff
@@ -75,7 +79,7 @@ def __init__(self, mode=None):
     def __str__(self):
         """String output for the config object
         """
-        return yaml.dump(self._root_config, default_flow_style=False)
+        return yaml.dump(self._root_config, default_flow_style=False, indent=4)
```

Comment:
only the travis file is indent=2, all pipeline definitions and other YAMLs in our code base are 4.